packages/framework/fluid-static/src/utils.ts

```diff
@@ -23,8 +23,8 @@ export type InternalDataObjectClass<T extends IFluidLoadable> = DataObjectClass<
 /**
  * Runtime check to determine if a class is a DataObject type
  */
-export const isDataObjectClass = (obj: any): obj is InternalDataObjectClass<IFluidLoadable> => {
-	const maybe: Partial<InternalDataObjectClass<IFluidLoadable>> | undefined = obj;
+export const isDataObjectClass = (obj: unknown): obj is InternalDataObjectClass<IFluidLoadable> => {
+	const maybe = obj as Partial<InternalDataObjectClass<IFluidLoadable>>;
```

Comment:
Seems like `Partial<>` handles the case of an undefined object too, correct?

Reply:
No, I don't believe it does. It would probably be better to cast this to `Partial<Foo> | undefined>`. I'll update.